### ISSUE TITLE
Any normalization required for the input image?

### ISSUE THREAD

**Issue by SeanGuo063** [NVlabs/RADIO#14]

**Comment by gheinrich** [comment]
Hello, the RADIO models expect their inputs to be in the range of  `[0, 1]`.

Normalization is taking place within the `forward()` method of the model in the `input_conditioner` module. There are examples in the `/examples` folder if you wish to take a look.

If you need to integrate the RADIO model within an existing pre-processing pipeline that already normalizes inputs to zero mean, unit variance, then one trick you can apply is to overwrite the input conditioner in the model and make it an  identity layer:
```
radio_model.input_conditioner = torch.nn.Identity()
```

**Comment by SeanGuo063** [comment]
That's very helpful. Thank you very much!
